builder/build.go

```diff
@@ -95,7 +95,7 @@ func createBuildTemplate(functionName string, handler string, language string) s
 	return tempPath
 }
 
-func buildFlagSlice(nocache bool, squash bool, httpProxy string, httpsProxy string, buildArgMap map[string]string) []string {
+func buildFlagSlice(nocache bool, squash bool, httpProxy string, httpsProxy string, buildArgMap map[string]string, buildOption string, language string) []string {
```

Comment:
The `buildFlagSlice()` signature could remain the same if the `buildOption` is converted to BuildArgs earlier.